assets/section-footer.css

```
@@ -337,7 +356,7 @@
 
 .footer-block__details-content .list-menu__item--link,
 .copyright__content a {
-  color: rgba(var(--color-foreground), 0.75);
+  color: black;
```

```suggestion
  color: rgba(var(--color-foreground), 1);
```

We should use a CSS variable for this, by finding the appropriate variable and changing it. This causes an issue in that when we change the color palette in the editor, this color will not change appropriately. It works for right now, but limits what the admin can do later.